### ISSUE TITLE
Feature requrst - Tessellation+stereoscopic mode

### ISSUE THREAD

**Issue by SpinAttack** [gonetz/GLideN64#1319]

**Comment by gonetz** [comment]
please provide description, what you expect from these features.

**Comment by fzurita** [comment]
I would imagine allowing the emulator to use Tessellation textures to introduce additional polygon detail to games.

Stereoscopic mode I would imagine means repeating the same image twice side by side to allow the plugin to be used with VR headsets.

**Comment by SpinAttack** [comment]
👍

**Comment by mudlord** [comment]
Point normal tessellation is an abysmal idea.

**Comment by fzurita** [comment]
I agree with that. It would have to be done in a similar fashion to texture packs.

**Comment by MatthewHunter1983** [comment]
tessellation sounds like a cool idea but I could see that totally distorting some models with no good result. maybe if we could select pieces of geometry to apply it to (similar to the 3dnes emulator where you can pause the game and edit the depth of the 3d layers). just an idea hope that doesn't detract from the request for stereoscopy, please do that first :) (for a 3dtv format, not VR. or just both since it would probably be the same method of injection)

**Comment by fzurita** [comment]
There was an initial attempt at adding 3D support here. It only works in Android right now though.:

https://github.com/gonetz/GLideN64/pull/1552

**Comment by MatthewHunter1983** [comment]
sooo cool! guess I could try on my android since it has hdmi output. but it's VR format? what is the most recent version that had 3d support?

I just want side by side or interlaced for 3dtv. a port to windows version would be so amazing

**Comment by fzurita** [comment]
Yeah, it's VR but it could easily be adapted to SBS 3D.
